test/e2e/accounts/snap-account-eth-swap.spec.ts

```diff
@@ -32,7 +32,6 @@ describe('Snap Account - Swap', function () {
     await withFixtures(
       {
         fixtures: new FixtureBuilder().build(),
-        ganacheOptions: defaultGanacheOptions,
```

Comment:
in all the specs where we were using defaultGanacheOptions, we can now remove this, as they default options are applied already by default